Form-Controls/index.html

```diff
@@ -1,27 +1,49 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>My form exercise</title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-  </head>
-  <body>
-    <header>
-      <h1>Product Pick</h1>
-    </header>
-    <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
-      </form>
-    </main>
-    <footer>
-      <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
-    </footer>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+  <body>
+    <header>
+      <h1>Customers'T-shirt Pick</h1>
+    </header>
+    <main>
+      <form>
+      <div>
+        <label for="username"> Enter username:</label>
+        <input type= "text" id="username" name="username" placeholder= "Username" required  autocomplete="off"/><br><br>   
+      </div>
+      <div>
+        <label for="email">Enter your email:</label>
+        <input type="email" id="email" name="email" placeholder="Email" required autocomplete="off"/><br><br>
+      </div>
+      <div>
+        <label for="T-shirtcolour">Choose a colour:</label>
+        <select id="T-shirtcolour" name="T-shirtcolour">

```

Comment:
If we declare a `<select>` element to be `required` (like we did in the `<input>` elements further up), then it means that a non-empty value must be selected. It's also usual to give an option at the beginning of your list of options, in order to show a message to the user that they need to select an option - otherwise they might accidentally pick the first one that shows without meaning to! You could do this with for example:

```html
<option value="">Colour</option>
```

Notice that the value is empty, which means that a user cannot select it, but it will show first.